src/generated/resources/openapi.json

```diff
@@ -1613,12 +1613,12 @@
             "enum" : [ "UP", "DOWN" ],
             "type" : "string"
           },
+          "name" : {
+            "type" : "string"
+          },
```

Comment:
The OpenAPI spec generator is finicky and non-deterministic w/r/t ordering of the `HealthCheck` type. Nevertheless, it'd be good to avoid changes like these.

Reply:
Will restore from main.